src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,10 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comment = [] }) => (
+  <div className="CommentList">
+    {comment.map(com => (
+      com === [] ? null : <CommentInfo com={com} key={com.id} />
```

Comment:
```suggestion
      !com.length ? null : <CommentInfo com={com} key={com.id} />
```

Reply:
Consider using descriptive names for your props (comment instead of com, for example)